master_mapping.py

```diff
@@ -572,5 +580,16 @@
         "name": "Wheel Buttons",
         "units": "",
     },
+    105: {
+        "name": "MPU Mode State",
+        "units": ""
```

Comment:
Food for thought, we might want to add in an enumerator in ner_processing to represent the state rather than decoding magic numbers. It might be beneficial for the user to have this type be an enum of the possible states, and have that data structure located somewhere within NER processing so that they can pull from it if need be.